src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -162,6 +164,19 @@
                         }
                     })
                 }
+                this.validateParams()
+            },
+            validateParams() {
+                const validValues = this.parameterValues.filter(param => (param.value))
```

Comment:
```suggestion
                const validValues = this.parameterValues.filter(param => param.value)
```
The brackets here are unnecessary and slightly confusing - they make the result look like a tuple